sdk/appcenter-storage/src/main/java/com/microsoft/appcenter/storage/TokenManager.java

```diff
@@ -0,0 +1,59 @@
+package com.microsoft.appcenter.storage;
+
+import com.microsoft.appcenter.utils.storage.SharedPreferencesManager;
+import com.google.gson.Gson;
+import com.microsoft.appcenter.storage.models.TokenResult;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Date;
+import java.util.List;
+
+public class TokenManager {
+
+    private final Gson gson;

```

Comment:
We are going to have a static `Gson` object shared within the module. For now, keep it as is, but add a _TODO_ in the comments to so that I replace it with the new reference before I merge my changes in.